### PR TITLE
Tpv launch

### PR DESCRIPTION
**Pull request by saxtot**
Simply adds a launch file for the TPV controller to markhor_bringup.

### CODE REVIEW

File: markhor_bringup/launch/tpv_controller.launch [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<launch>
+    <node name="microcontroller_command_node"        pkg="rosserial_python"      type="serial_node.py">
```
Comment (Lux-Ludo) [inline markhor_bringup/launch/tpv_controller.launch:2]: Are the spaces necessary ? And if not, I think we should follow the same standard that we did for the other launch files.

[the remark this answers is not in the document]
Comment (saxtot) [reply markhor_bringup/launch/tpv_controller.launch]: Absolutely not necessary, they have been removed.

**Review by lvanasse — APPROVED** [review]

**Comment by lvanasse** [comment]
Ah maybe, do you want to add in the README a new bullet point for this dependency? Like we did for the e-sport and such.

**Comment by saxtot** [comment]
> Ah maybe, do you want to add in the README a new bullet point for this dependency? Like we did for the e-sport and such.

Correct me if I'm wrong, but wouldn't a dependency be absolutely required for the package to work? For example, the launch files include most of the multiplicators, however, the robot still launches as the nodes simply fail or don't work. So, we never considered Markhor dependent to them.
Likewise, in the meager amount of tests I ran, the node working wasn't necessary for the rest of the ROS environment.

**Comment by Lux-Ludo** [comment]
Well, in that case, if we launch the `markhor_base.launch` launch file without the microcontroller node it will likely produce error, although it might not fail.

It kind of the same situation as for this PR that Myriam did a while back : https://github.com/clubcapra/markhor/pull/24 

And you're right, maybe dependancy is not the right term. Although my intension is to be clear about what needs to be present in the workspace for the launch file to work without error. I'm open to idea if you have a way to accomplish this without too much trouble.

**Comment by lvanasse** [comment]
Well, in that case, if we launch the `markhor_base.launch` launch file without the microcontroller node it will likely produce error, although it might not fail.

It kind of the same situation as for this PR that Myriam did a while back : https://github.com/clubcapra/markhor/pull/24

And you're right, maybe dependency is not the right term. Although, my intention is to be clear about what needs to be present in the workspace for the launch file to work without error. I'm open to ideas if you have a way to accomplish this without too much trouble.

**Comment by saxtot** [comment]
What would you think of  a simple "if" statement in the launch file? Where if the package is not found, it is printed once in the console without it necessarily being an error.

**Comment by benmalenfant** [comment]
Looking good!👍🏻

**Review by benmalenfant — DISMISSED** [review]

**Comment by lvanasse** [comment]
> What would you think of a simple "if" statement in the launch file? Where if the package is not found, it is printed once in the console without it necessarily being an error.

Humm I think I prefer an error. But that's alright if you want to do it this way.

**Comment by saxtot** [comment]
I think I understand what you mean then.
We could leave the code as is and add a section in the README with the packages that are expected to be present upon normal use, but aren't entirely necessary to launch markhor.

**Comment by lvanasse** [comment]
> I think I understand what you mean then. We could leave the code as is and add a section in the README with the packages that are expected to be present upon normal use, but aren't entirely necessary to launch markhor.

Yup, I think that would be more clear 👍

**Review by lvanasse — APPROVED** [review]

**Pull request merged**